modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -27,96 +31,183 @@ The repository must be accessible from the hub cluster and be defined as a sourc
 
 .Procedure
 
-. Create a `ConfigMap` resource that describes the NICs for a group of hosts. For example:
+. Create three `ConfigMap` CRs that contain the group and site configuration. 
++
+.. Create  a `ConfigMap` CR named `group-hardware-types-configmap` to hold the hardware-specific configuration. For example:
 +
 [source,yaml]
 ----
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: sriovdata
-  namespace: ztp-site
+  name: group-hardware-types-configmap
+  namespace: ztp-group
   annotations:
     argocd.argoproj.io/sync-options: Replace=true <1>
 data:
-  example-sno-du_fh-numVfs: "8"
-  example-sno-du_fh-pf: ens1f0
-  example-sno-du_fh-priority: "10"
-  example-sno-du_fh-vlan: "140"
-  example-sno-du_mh-numVfs: "8"
-  example-sno-du_mh-pf: ens3f0
-  example-sno-du_mh-priority: "10"
-  example-sno-du_mh-vlan: "150"
+  # SriovNetworkNodePolicy.yaml
+  hardware-type-1-sriov-node-policy-pfNames-1: "[\"ens5f0\"]"
+  hardware-type-1-sriov-node-policy-pfNames-2: "[\"ens7f0\"]"
+  # PerformanceProfile.yaml
+  hardware-type-1-cpu-isolated: "2-31,34-63"
+  hardware-type-1-cpu-reserved: "0-1,32-33"
+  hardware-type-1-hugepages-default: "1G"
+  hardware-type-1-hugepages-size: "1G"
+  hardware-type-1-hugepages-count: "32"
 ----
 <1> The `argocd.argoproj.io/sync-options` annotation is required only if the `ConfigMap` is larger than 1 MiB in size.
+
+.. Create  a `ConfigMap` CR named `group-zones-configmap` to hold the regional configuration. For example:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: group-zones-configmap
+  namespace: ztp-group
+data:
+  # ClusterLogForwarder.yaml
+  zone-1-cluster-log-fwd-outputs: "[{\"type\":\"kafka\", \"name\":\"kafka-open\", \"url\":\"tcp://10.46.55.190:9092/test\"}]"
+  zone-1-cluster-log-fwd-pipelines: "[{\"inputRefs\":[\"audit\", \"infrastructure\"], \"labels\": {\"label1\": \"test1\", \"label2\": \"test2\", \"label3\": \"test3\", \"label4\": \"test4\"}, \"name\": \"all-to-default\", \"outputRefs\": [\"kafka-open\"]}]"
+----
+
+.. Create a `ConfigMap` CR named `site-data-configmap` to hold the site-specific configuration. For example:
++
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: site-data-configmap
+  namespace: ztp-group
+data:
+  # SriovNetwork.yaml
+  du-sno-1-zone-1-sriov-network-vlan-1: "140"
+  du-sno-1-zone-1-sriov-network-vlan-2: "150"
+----
 +
 [NOTE]
```

Comment:
This note seems like it applies to the overall step, but right now it's nested under step 1c. If you mean this to be a note for the entire step, you could try these two options instead:

1. Moving this note above the second level substeps
2. (Not sure this will work but) You could try adding `--` in between lines 35-36, and in between lines 91-92. This will tell asciibinder to treat everything in that `--` block as if it was its own discrete block of asciidoc text, and thus the `+` on line 92 might be interpreted as a continuation of the overall step and not step 1c. Not sure if it'll work though, maybe the `..` of the substeps won't be interpreted correctly

Up to you if either of these options interest you!

Reply:
Thanks for the suggestion @skopacz1 
I went for option 2 here and yes it worked